sbtc-core/src/signer/mod.rs

```diff
@@ -0,0 +1,119 @@
+/// An sBTC signer configuration
+pub mod config;
+
+use std::collections::HashMap;
+
+use crate::signer::config::Config;
+use crate::SBTCError;
+use bitcoin::{Network, PrivateKey, PublicKey, Transaction as BitcoinTransaction};
+use p256k1::ecdsa;
+use url::Url;
+use wsts::{bip340::SchnorrProof, common::Signature};
+
+#[derive(Default, Clone, Debug)]
+pub struct SignersData {
+    pub signers: HashMap<u32, ecdsa::PublicKey>,
+    pub vote_shares: HashMap<u32, ecdsa::PublicKey>,
```

Comment:
What's the difference between signers and vote shares here?

Reply:
Maybe this should map keys to vote shares (vice versa)?

Reply:
Signers maps signer ids to their public keys. Vote shares map key ids to public keys.